### ISSUE TITLE
Undefined index Notices

### ISSUE THREAD

**Issue by Remzi1993** [second-2None/service-area-postcode-checker#1]
Hi @second-2None 

I got three notices about Undefined index, see 

```
Notice: Undefined index: title-display in htdocs\wp-content\plugins\service-area-postcode-checker\options.php on line 328

Notice: Undefined index: title-display in htdocs\wp-content\plugins\service-area-postcode-checker\options.php on line 344

Notice: Undefined index: title-display in htdocs\wp-content\plugins\service-area-postcode-checker\options.php on line 360
```

I think it's about:
- **$display_option['title-display']** on line 328
- **$display_option['class-display']** on line 344
- **$display_option['class-bullet']** on line 366

Kind regards
@Remzi1993

**Comment by Kane-R-G** [comment]
Ok great thanks for letting me know I'll take a look and see what might be causing it.

**Comment by Kane-R-G** [comment]
This should be fixed in the 2.0.8 update.

**Comment by Remzi1993** [comment]
@second-2None Thank you very much!
